RockWeb/Blocks/Core/LocationDetail.ascx.cs

```diff
@@ -51,6 +51,9 @@ namespace RockWeb.Blocks.Core
     public partial class LocationDetail : RockBlock, IDetailBlock
     {
         private int? _personId = null;
+        private Location _location;
+        private RockContext _rockContext;
```

Comment:
Hi Mark @trydyingtolive , although you may have stumbled across this pattern (used in 9 _other_ blocks) it was primarily supposed to be for blocks that had ItemDataBound events.  (And I know some of those blocks are in violation of that rule but we've been slowly moving away from this pattern.)  Would you be willing to resubmit this with just the straight-forward fix?